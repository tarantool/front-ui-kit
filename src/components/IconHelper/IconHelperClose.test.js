import React from 'react';
import renderer from 'react-test-renderer';
import { IconHelperClose } from './index';

it('IconHelperClose renders correctly', () => {
  const tree = renderer.create(
    <>
      <IconHelperClose />
    </>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

