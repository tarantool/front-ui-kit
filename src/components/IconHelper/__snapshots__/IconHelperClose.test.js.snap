// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconHelperClose renders correctly 1`] = `
<svg
  className="css-16d6do5"
  viewBox="0 0 16 16"
>
  <use
    xlinkHref="#close"
  />
</svg>
`;
